### ISSUE TITLE
BUG: Treatment list default visibility (wrong value assigned)

### ISSUE THREAD

**Issue by HartLarsson** [diegolamonica/EUCookieLaw3#4]
By selecting:
![image](https://user-images.githubusercontent.com/16102074/40621413-f0146aa2-629c-11e8-87c4-57b3ce089347.png)

The result code is not right:
![image](https://user-images.githubusercontent.com/16102074/40621445-062bf3c8-629d-11e8-9713-c36bc155f9ed.png)

the value must be true:

"dialogCookieGroupsListDefaultIsHidden":true

**Comment by mariobarretta** [comment]
Anche assegnando il valore `true` il banner della scelta continua ad apparire per esteso.

A seguire il codice col valore corretto ( provato con Chrome  Versione 67.0.3396.62 (Build ufficiale) (a 64 bit) e Versione 67.0.3396.62 (Build ufficiale) beta (a 64 bit) e Firefox 60.0.1 64 bit


```html
<!-- EUCookieLaw 3.0 - BEGIN -->
<script src="https://marianicucine.it/wp-content/plugins/EUCookieLaw3-wp-master/scripts/eucookielaw3.min.js"></script>
<script type="text/javascript">
// <![CDATA[
// Generated with EUCookieLaw Configuration Builder on Fri Jun 01 2018
// Builder version 20180528.1

var cookieLaw = new EUCookieLaw({"agreeMethod":2,"cookieEnabled":false,"dialogCookieGroupsListDefaultIsHidden":true});

/* Dialog properties */
cookieLaw.updateDialogProperty("header-container", "content" ,"Informativa sull'uso dei cookie");
cookieLaw.updateDialogProperty("body-text-content", "content" ,"In questo sito si utilizzano cookie di terze parti (prodotti da Twitter, Facebook, Google e altri servizi) per offrire un'esperienza di navigazione migliore. Tali cookie sono bloccati fino al tuo consenso ma potrebbero pregiudicare l'esperienza del sito se non autorizzati. Consulta l'<a href=\"https://marianicucine.it/privacy\">Informativa Estesa</a>. per maggiori dettagli.");
cookieLaw.updateDialogProperty("review-button", "content" ,"Guarda i consensi");
cookieLaw.updateDialogProperty("review-button", "classes" ,"Fatto");
cookieLaw.updateDialogProperty("close-button", "content" ,"Chiudi");
/* Blocking rules */
/* facebook*/
cookieLaw.registerGroup( "facebook", "Facebook", "Facebook Cookies", false );
cookieLaw.registerURL( "facebook", [".facebook.net",".facebook.com",".facebook.it"]); 
/* all-google-services*/
cookieLaw.registerGroup( "all-google-services", "All Google Services", "Google Cookies", false );
cookieLaw.registerURL( "all-google-services", [".google.it",".google.com",".googleapis.com",".googlesyndication.com",".doubleclick.net"]); 
/* google-adsense*/
cookieLaw.registerGroup( "google-adsense", "Google Adsense", "AdSense Cookies", false );
cookieLaw.registerURL( "google-adsense", [".googlesyndication.com/pagead/"]); 
/* google-analytics*/
cookieLaw.registerGroup( "google-analytics", "Google Analytics", "Analytics Cookies", false );
cookieLaw.registerURL( "google-analytics", [".google-analytics.com"]); 
/* google-fonts*/
cookieLaw.registerGroup( "google-fonts", "Google Fonts", "Google Fonts Cookies", false );
cookieLaw.registerURL( "google-fonts", [".fonts.googleapis.com"]); 
/* google-maps*/
cookieLaw.registerGroup( "google-maps", "Google Maps", "Google Maps Cookies", false );
cookieLaw.registerURL( "google-maps", [".maps.google.com",".maps.googleapis.com",".www.google.com/maps",".www.google.it/maps"]); 
/* instagram*/
cookieLaw.registerGroup( "instagram", "Instagram", "instagram Cookies", false );
cookieLaw.registerURL( "instagram", [".instagram.com",".cdninstagram.com"]); 
/* linkedin*/
cookieLaw.registerGroup( "linkedin", "LinkedIn", "Linkedin Cookies", false );
cookieLaw.registerURL( "linkedin", [".linkedin.com"]); 
/* pinterest*/
cookieLaw.registerGroup( "pinterest", "Pinterest", "Pinteres Cookies", false );
cookieLaw.registerURL( "pinterest", [".pinterest.com"]); 
/* twitter*/
cookieLaw.registerGroup( "twitter", "Twitter", "Twitter Cookies", false );
cookieLaw.registerURL( "twitter", [".twitter.com",".twitterfeed.com"]); 
/* vimeo*/
cookieLaw.registerGroup( "vimeo", "Vimeo", "Vimeo Cookies", false );
cookieLaw.registerURL( "vimeo", [".vimeo.com"]); 
/* youtube*/
cookieLaw.registerGroup( "youtube", "YouTube", "YouTube Cookies", false );
cookieLaw.registerURL( "youtube", [".youtube-nocookie.com",".youtube.com"]); 
cookieLaw.run();
/* Configuration info
/EUCookieLawID::
eyJhZ3JlZU1ldGhvZCI6MiwiY29va2llRW5hYmxlZCI6ZmFsc2UsImRpYWxvZ0Nv
b2tpZUdyb3Vwc0xpc3REZWZhdWx0SXNIaWRkZW4iOmZhbHNlLCJkaWFsb2dQcm9w
ZXJ0aWVzIjp7ImhlYWRlci1jb250YWluZXIvY29udGVudCI6IkluZm9ybWF0aXZh
IHN1bGwndXNvIGRlaSBjb29raWUiLCJib2R5LXRleHQtY29udGVudC9jb250ZW50
IjoiSW4gcXVlc3RvIHNpdG8gc2kgdXRpbGl6emFubyBjb29raWUgZGkgdGVyemUg
cGFydGkgKHByb2RvdHRpIGRhIFR3aXR0ZXIsIEZhY2Vib29rLCBHb29nbGUgZSBh
bHRyaSBzZXJ2aXppKSBwZXIgb2ZmcmlyZSB1bidlc3BlcmllbnphIGRpIG5hdmln
YXppb25lIG1pZ2xpb3JlLiBUYWxpIGNvb2tpZSBzb25vIGJsb2NjYXRpIGZpbm8g
YWwgdHVvIGNvbnNlbnNvIG1hIHBvdHJlYmJlcm8gcHJlZ2l1ZGljYXJlIGwnZXNw
ZXJpZW56YSBkZWwgc2l0byBzZSBub24gYXV0b3JpenphdGkuIENvbnN1bHRhIGwn
PGEgaHJlZj1cImh0dHBzOi8vbWFyaWFuaWN1Y2luZS5pdC9wcml2YWN5XCI+SW5m
b3JtYXRpdmEgRXN0ZXNhPC9hPi4gcGVyIG1hZ2dpb3JpIGRldHRhZ2xpLiIsInJl
dmlldy1idXR0b24vY29udGVudCI6Ikd1YXJkYSBpIGNvbnNlbnNpIiwicmV2aWV3
LWJ1dHRvbi9jbGFzc2VzIjoiRmF0dG8iLCJjbG9zZS1idXR0b24vY29udGVudCI6
IkNoaXVkaSJ9LCJibG9ja2VkR3JvdXBzIjp7IkZhY2Vib29rIjp7ImRlc2NyaXB0
aW9uIjoiRmFjZWJvb2sgQ29va2llcyIsImVuYWJsZWRCeURlZmF1bHQiOmZhbHNl
LCJrZXlzIjpbIi5mYWNlYm9vay5uZXQiLCIuZmFjZWJvb2suY29tIiwiLmZhY2Vi
b29rLml0Il19LCJBbGwgR29vZ2xlIFNlcnZpY2VzIjp7ImRlc2NyaXB0aW9uIjoi
R29vZ2xlIENvb2tpZXMiLCJlbmFibGVkQnlEZWZhdWx0IjpmYWxzZSwia2V5cyI6
WyIuZ29vZ2xlLml0IiwiLmdvb2dsZS5jb20iLCIuZ29vZ2xlYXBpcy5jb20iLCIu
Z29vZ2xlc3luZGljYXRpb24uY29tIiwiLmRvdWJsZWNsaWNrLm5ldCJdfSwiR29v
Z2xlIEFkc2Vuc2UiOnsiZGVzY3JpcHRpb24iOiJBZFNlbnNlIENvb2tpZXMiLCJl
bmFibGVkQnlEZWZhdWx0IjpmYWxzZSwia2V5cyI6WyIuZ29vZ2xlc3luZGljYXRp
b24uY29tL3BhZ2VhZC8iXX0sIkdvb2dsZSBBbmFseXRpY3MiOnsiZGVzY3JpcHRp
b24iOiJBbmFseXRpY3MgQ29va2llcyIsImVuYWJsZWRCeURlZmF1bHQiOmZhbHNl
LCJrZXlzIjpbIi5nb29nbGUtYW5hbHl0aWNzLmNvbSJdfSwiR29vZ2xlIEZvbnRz
Ijp7ImRlc2NyaXB0aW9uIjoiR29vZ2xlIEZvbnRzIENvb2tpZXMiLCJlbmFibGVk
QnlEZWZhdWx0IjpmYWxzZSwia2V5cyI6WyIuZm9udHMuZ29vZ2xlYXBpcy5jb20i
XX0sIkdvb2dsZSBNYXBzIjp7ImRlc2NyaXB0aW9uIjoiR29vZ2xlIE1hcHMgQ29v
a2llcyIsImVuYWJsZWRCeURlZmF1bHQiOmZhbHNlLCJrZXlzIjpbIi5tYXBzLmdv
b2dsZS5jb20iLCIubWFwcy5nb29nbGVhcGlzLmNvbSIsIi53d3cuZ29vZ2xlLmNv
bS9tYXBzIiwiLnd3dy5nb29nbGUuaXQvbWFwcyJdfSwiSW5zdGFncmFtIjp7ImRl
c2NyaXB0aW9uIjoiaW5zdGFncmFtIENvb2tpZXMiLCJlbmFibGVkQnlEZWZhdWx0
IjpmYWxzZSwia2V5cyI6WyIuaW5zdGFncmFtLmNvbSIsIi5jZG5pbnN0YWdyYW0u
Y29tIl19LCJMaW5rZWRJbiI6eyJkZXNjcmlwdGlvbiI6IkxpbmtlZGluIENvb2tp
ZXMiLCJlbmFibGVkQnlEZWZhdWx0IjpmYWxzZSwia2V5cyI6WyIubGlua2VkaW4u
Y29tIl19LCJQaW50ZXJlc3QiOnsiZGVzY3JpcHRpb24iOiJQaW50ZXJlcyBDb29r
aWVzIiwiZW5hYmxlZEJ5RGVmYXVsdCI6ZmFsc2UsImtleXMiOlsiLnBpbnRlcmVz
dC5jb20iXX0sIlR3aXR0ZXIiOnsiZGVzY3JpcHRpb24iOiJUd2l0dGVyIENvb2tp
ZXMiLCJlbmFibGVkQnlEZWZhdWx0IjpmYWxzZSwia2V5cyI6WyIudHdpdHRlci5j
b20iLCIudHdpdHRlcmZlZWQuY29tIl19LCJWaW1lbyI6eyJkZXNjcmlwdGlvbiI6
IlZpbWVvIENvb2tpZXMiLCJlbmFibGVkQnlEZWZhdWx0IjpmYWxzZSwia2V5cyI6
WyIudmltZW8uY29tIl19LCJZb3VUdWJlIjp7ImRlc2NyaXB0aW9uIjoiWW91VHVi
ZSBDb29raWVzIiwiZW5hYmxlZEJ5RGVmYXVsdCI6ZmFsc2UsImtleXMiOlsiLnlv
dXR1YmUtbm9jb29raWUuY29tIiwiLnlvdXR1YmUuY29tIl19fX0=
::EUCookieLawID/*/
//]]></script>
<!-- EUCookieLaw 3.0 - END -->
```

**Comment by diegolamonica** [comment]
@HartLarsson, thanks for th feedback... This issue is solved with [the new bulder v 20180625.1](http://diegolamonica.info/tools/eucookielaw/builder/) available since now!
